include/SubWindow.h

```diff
@@ -43,16 +43,40 @@ class QWidget;
 class EXPORT SubWindow : public QMdiSubWindow
 {
 	Q_OBJECT
+    Q_PROPERTY(QBrush activeColor READ activeColor WRITE setActiveColor)
+    Q_PROPERTY(QColor textShadowColor READ textShadowColor WRITE setTextShadowColor)
+    Q_PROPERTY(QColor borderColor READ borderColor WRITE setBorderColor)
+
 public:
-	SubWindow(QWidget *parent=NULL, Qt::WindowFlags windowFlags=0);
+	SubWindow(QWidget *parent = NULL, Qt::WindowFlags windowFlags = 0);
 	// same as QWidet::normalGeometry, but works properly under X11 (see https://bugreports.qt.io/browse/QTBUG-256)
 	QRect getTrueNormalGeometry() const;
+	QBrush activeColor() const;
+	QColor textShadowColor() const;
+	QColor borderColor() const;
+	void setActiveColor(const QBrush & b);
+	void setTextShadowColor(const QColor &c);
+	void setBorderColor(const QColor &c);
+
 protected:
 	// hook the QWidget move/resize events to update the tracked geometry
-	virtual void moveEvent(QMoveEvent * event);
-	virtual void resizeEvent(QResizeEvent * event);
+	virtual void moveEvent( QMoveEvent * event );
+	virtual void resizeEvent( QResizeEvent * event );
+	virtual void paintEvent( QPaintEvent * pe );
+	
 private:
+    QPushButton * m_closeBtn;
+    QPushButton * m_minimizeBtn;
+    QPushButton * m_maximizeBtn;
```

Comment:
Mind the indentation over here please, looks like you have mixed tabs and spaces.